app/create.py

```diff
@@ -41,8 +42,11 @@ def days_to_date(days):
 class Handler(BaseHandler):
     def get(self):
         self.params.create_mode = True
-        profile_websites = [add_profile_icon_url(website, self)
-                for website in self.config.profile_websites or []]
+        profile_websites = [
+            # add_profile_icon_url is going to modify website, so we make a copy
+            # of the config setting to avoid modifying the original.
+            add_profile_icon_url(website, self)
```

Comment:
Can be a TODO, but maybe add_profile_icon_url() should rather perform deep copy internally so that it doesn't cause modification to its parameter?

Reply:
Done.